src/homepage/homepage.html

```diff
@@ -45,14 +95,19 @@
     }
 
     #activities div:nth-child(2n + 1) {
-      background-color: var(--tertiary-color);
+      background-color: var(--primary-color);
```

Comment:
It does colors in a different pattern to start with, but imo the tertiary color is so badly chosen that sticking to two colors looked way better. Your pick tho